main.tf

```diff
@@ -13,28 +13,34 @@ locals {
     }])
     scopes = tolist([])
   }])
-  database_users = length(var.database_users) == 0 ? local.database_users_default : var.database_users
+  database_users = length(var.database_users) == 0 && !var.use_existing_project ? local.database_users_default : var.database_users
```

Comment:
curious about this change?

Reply:
Yes, I should have given a bit of context. I found that the following problem, imagine this scenario:
A user calls the module with an existing project and does not provide a database user (the module is going to create a default database user).
Then, the user calls the module again for the same project and again does not provide a database user. The problem here is that the module is going to try to create again a default database user (with the same name as the previous), which gives an error or is something we don't want to happen.
The work-around I decided to do is to only create a default database user if the module creates the project and no user is provided. If an existing project is provided, no default user will be created even if no user is provided.